.github/workflows/build-main.yml

```diff
@@ -21,16 +21,12 @@ jobs:
     - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
       with:
         persist-credentials: false
-    - name: Fetch latest release version
-      uses: reloc8/action-latest-release-version@b8d6337f30390558e7874a044d6a3c1314314bab # 1.0.0
-      id: fetch-latest-release
 
     - name: Set release tag and short SHA
       run: |
-        echo "RELEASE_TAG=${RELEASE_TAG}" >> "${GITHUB_ENV}"
+        git fetch --tags
+        echo "RELEASE_TAG=$(git describe --abbrev=0 --tags)" >> "${GITHUB_ENV}"
```

Comment:
It seems smart to get ride of this since it hasn't been updated in 4 years, do you think we'd need the below line?
```
git fetch --prune --unshallow || true
```
Or not needed in our case since we have a fresh clone here, and the above would always just be `true`?

Reply:
Just wanted to make sure we were thinking the same thing.